src/ring_theory/int/basic.lean

```diff
@@ -86,8 +86,33 @@ end⟩
 instance : unique_factorization_monoid ℕ :=
 ⟨λ _, nat.irreducible_iff_prime⟩
 
+instance unique_units : unique (units ℕ) :=
+{ default := 1, uniq := nat.units_eq_one }
+
+instance unique_add_units : unique (add_units ℕ) :=
+{ default := 0, uniq := nat.add_units_eq_zero }
+
+@[simp] lemma normalize_eq (n : ℕ) : normalize n = n := n.mul_one
+
 end nat
 
+/-- `ℕ` is a gcd_monoid. -/
+instance : gcd_monoid ℕ :=
```

Comment:
Does `ring_theory/nat/basic` exist? This doesn't seem like the most natural place for this result. But I understand that these basic files are a bit of a tangle, where we need to get ourselves of the ground... so if it doesn't naturally fit anywhere else, I don't think it's worth creating a new file for just this thing.